test/smoke/packages/contour/1.18.2/contour-test.sh

```diff
@@ -15,19 +15,20 @@ then
     version=$(tanzu package available list contour.community.tanzu.vmware.com | tail -n 1 | awk '{print $2}')
     tanzu package install contour --package-name contour.community.tanzu.vmware.com -f "${MY_DIR}"/contour-values.yaml --version "${version}"
 fi
-sleep 15s
 
 # Providing prerequisite 
-kubectl create namespace contour-example-workload
-kubectl create deployment nginx-example --image nginx --namespace contour-example-workload
-kubectl create service clusterip nginx-example --tcp 80:80 --namespace contour-example-workload
+NAMESPACE_SUFFIX=${RANDOM}
+NAMESPACE="contour-${NAMESPACE_SUFFIX}"
+kubectl create ns ${NAMESPACE}
+kubectl create deployment nginx-example --image nginx --namespace ${NAMESPACE}
+kubectl create service clusterip nginx-example --tcp 80:80 --namespace ${NAMESPACE}
```

Comment:
I have noticed some machines require root access to bind to reserved ports like `80` or anything less than `1024`. Something to note